tensorflow/java/src/main/java/org/tensorflow/Output.java

```diff
@@ -57,6 +60,30 @@ public Output asOutput() {
     return this;
   }
 
+  @Override
+  public int hashCode() {
+    return operation.hashCode() + 37 * index + 17;
```

Comment:
This is a [suggested approach](https://stackoverflow.com/questions/113511/best-implementation-for-hashcode-method) via Josh Bloch and avoids the implicit array object creation for the varargs, but either way is good enough for me; I'll change to using the utils.